fetch/api/response/multi-globals/current/current.html

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE html>
```

Comment:
We could, but I think it's clearer to have the files separated by test, instead of trying to be conservative about the number of files. There is no real advantage in minimizing the number of files, in my opinion.